code/modules/mob_spawn/corpses/mining_corpses.dm

```diff
@@ -274,3 +336,25 @@
 		/obj/item/stack/sheet/runed_metal = 15,
 		)
 	r_pocket = /obj/item/clothing/glasses/hud/health/night/cultblind
+
+/datum/outfit/consumed_heremoth
+	name = "Legion-Consumed Tribal Mothman"
+	suit = /obj/item/clothing/suit/hooded/cultrobes/eldritch
+	head = /obj/item/clothing/head/hooded/cult_hoodie/eldritch
+
+/datum/outfit/consumed_heremoth/pre_equip(mob/living/carbon/human/moth, visualsOnly = FALSE)
+	if(!visualsOnly)
+		moth.set_species(/datum/species/moth)
+	if(prob(70))
+		glasses = /obj/item/clothing/glasses/blindfold
+	if(prob(70))
+		belt = pick(list(
+			/obj/item/flashlight/lantern = 1,
+			/obj/item/toy/plush/moth = 1,
+		))
```

Comment:
```suggestion
		if(prob(50))
			belt = /obj/item/flashlight/lantern
		else
			l_pocket = /obj/item/toy/plush/moth
```
Looks like you can't equip a moth plus to the belt, but you could put it in the pocket instead.